.babelrc

```diff
@@ -16,7 +16,8 @@
     "test": {
       "plugins": [
         "transform-class-properties",
-        "transform-object-assign"
+        "transform-object-assign",
+        "react-hot-loader/babel"
```

Comment:
Is this working?  I think this only adds it if `NODE_ENV=test`

We might want to make a separate `"development"` property for the dev plugins like this one and see how that works